doc/how_to_guides/pick_ik/config/kinematics_pick_ik.yaml

```diff
@@ -0,0 +1,11 @@
+panda_arm:
+  kinematics_solver: pick_ik/PickIkPlugin
```

Comment:
It would be nice to add some helpful in-line comments here. People tend to copy-paste these example files into their own applications, so some in-line comments can be a helpful reminder as to what these parameters mean. See [this example](https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/how_to_guides/parallel_planning/config/parallel_planning_moveit_cpp.yaml) configuration file from the parallel planning tutorial for an example of what in-line comments can look like.